spec/abstract_controller_base_spec.rb

```diff
@@ -165,15 +207,32 @@ def tmpdir
             )
           end
 
+          it 'ignores the text template' do
+            expect(events).to_not include hash_including(
+              'event' => 'call',
+              'method_id' => 'render',
+              'render_template' => anything
+            )
+
+            expect(appmap['classMap']).to_not include hash_including(
+              'name' => 'views',
+              'children' => include(hash_including(
+                'name' => 'ViewTemplate',
+                'children' => include(hash_including(
+                  'name' => 'render',
+                  'type' => 'function',
+                  'location' => 'text template'
+                ))
+              ))
+            )
+          end
+
           it 'records and labels view rendering' do
             expect(events).to include hash_including(
               'event' => 'call',
               'thread_id' => Numeric,
-              'defined_class' => 'ActionView::Renderer',
-              'method_id' => 'render',
-              'path' => String,
-              'lineno' => Integer,
-              'static' => false
+              'defined_class' => 'inline_template',
```

Comment:
When rendering a template that's not a file, (e.g. it's a Ruby string), the class name `inline_template` is used.